docs/02_cleos/03_command-reference/wallet/create.md

```diff
@@ -7,4 +7,85 @@ None
 ## Options
 - `-n, --name` _TEXT_ - The name of the new wallet
 - `-f, --file` _TEXT_ - Name of file to write wallet password output to. (Must be set, unless "--to-console" is passed
-- `--to-console` - Print password to console
\ No newline at end of file
+- `--to-console` - Print password to console
+
+## Command
+cleos wallet create [OPTIONS]
+
+**Where**
+* [OPTIONS] = See Options in Command Usage section below. 
+
+**Note**: The arguments and options enclosed in square brackets are optional.
+
+## Description
+Create a new wallet locally.
+
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos wallet create` command:
+
+### Positionals:
+- none
+### Options
+`-h`, `--help` - Print this help message and exit
+`-n`, `--name` _TEXT_ - The name of the new wallet, if blank the name is `default`
+`-f`, `--file` _TEXT_ - Name of file to write wallet password output to. Must be set, unless "--to-console" is passed
+`--to-console` - Print password to console.
+
+## Requirements
+* Install the currently supported version of `cleos` and `keosd`.
+[[info | Note]]
+| The `cleos` tool and `keosd` wallet is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
```

Comment:
The cleos tool -> \`cleos\`

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.